<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDlg</class>
 <widget class="QDialog" name="ImportDlg">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>406</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import Schedule File Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="fileTxt"/>
     </item>
     <item>
      <widget class="QPushButton" name="fileBtn">
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGrp">
     <property name="title">
      <string>Choose what to import</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="activitiesCbx">
        <property name="text">
         <string>Activities</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="usersCbx">
        <property name="text">
         <string>Users</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mergeCbx">
     <property name="text">
      <string>Merge all the content (activities, users, scheduled activities)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Priority</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="currentRBtn">
        <property name="text">
         <string>Current file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="importedRBtn">
        <property name="text">
         <string>Imported File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="importBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="errorsGrp">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Errors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="errorListTxt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
